src/main/resources/permissions/mod-tlr.csv

```diff
@@ -20,3 +20,4 @@ dcb.ecs-request.transactions.post
 circulation.requests.allowed-service-points.get
 dcb.transactions.get
 dcb.transactions.put
+consortium-search.items.item.get
```

Comment:
You are calling mod-search in local tenant, so this permission should not be here. It should be in `modulePermissions` in ModuleDescriptor.json